packages/nextjs/rollup.config.js

```diff
@@ -0,0 +1,47 @@
+import resolve from '@rollup/plugin-node-resolve';
+import ts from 'rollup-plugin-typescript2';
+import path from 'path';
+import clientComponentPlugin from '../../scripts/rollup-plugins/clientComponentPlugin';
+import movePackageJsonPlugin from '../../scripts/rollup-plugins/movePackageJsonPlugin';
```

Comment:
remove rollup

Reply:
and use webpack?

Reply:
will be converted in the future along with other sdks (react/angular/vue)